langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaChatModelIT.java

```diff
@@ -0,0 +1,39 @@
+package dev.langchain4j.model.ollama;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.model.output.Response;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static dev.langchain4j.data.message.SystemMessage.systemMessage;
+import static dev.langchain4j.data.message.UserMessage.userMessage;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Disabled("needs Ollama running locally")
```

Comment:
There is no need in this any more, please see what awesome @eddumelendez did in [this PR](https://github.com/langchain4j/langchain4j/pull/315)